pkg/cluster/reconcile.go

```
@@ -244,6 +248,7 @@ func (c *Cluster) disasterRecovery(left etcdutil.MemberSet) error {
 		}
 	}
 	c.members = nil
+	// MAYBE: restore a self-hosted etcd cluster.
 	return c.restoreSeedMember()
```

Can we return an error here instead?
```
if c.spec.SelfHosted{
  return fmt.Errorf("unsupported cluster (%s) self hosted restore seed member", ...)
}
```
We can see the error from logs